script/utils/prepareRedirects.ts

```diff
@@ -37,13 +39,34 @@ const prepareRedirects = async(
         continue;
       }
 
+      const to = decodeURI(content.getUrl({ book: { slug: bookSlug }, page: { slug: page.slug } })) + (query || '');
+
       redirects.push({
         from: pathname,
-        to: decodeURI(content.getUrl({ book: { slug: bookSlug }, page: { slug: page.slug } })) + (query || ''),
+        to,
       });
+
+      if (!pathname.endsWith('/')) {
+        redirects.push({
+          from: `${pathname}/`,
+          to,
+        });
```

Comment:
do this with ```{from: `${pathname}/`, to: pathname}``` and it'll do two redirects, one from the slash to the no slash and one from the no slash to the new location.

this way any previous redirects for standard pages that have slash-to-no-slash redirects will have the same slash-to-no-slash redirects if the pages are subsequently moved.

not that the way you have it would be broken, but it would be better for the redirects for particular urls to not change because they're allowed to be indefinitely cached and it seems like it could be a source of unexpected behavior in the future to alter them